proxy/logging/LogAccess.cc

```diff
@@ -326,11 +328,11 @@ LogAccess::marshal_proxy_resp_content_type(char *buf)
 /*-------------------------------------------------------------------------
   -------------------------------------------------------------------------*/
 
-int
-LogAccess::marshal_proxy_resp_squid_len(char *buf)
-{
-  DEFAULT_INT_FIELD;
-}
+LOG_ACCESS_DEFAULT_FIELD(marshal_proxy_resp_squid_len, DEFAULT_INT_FIELD)
+LOG_ACCESS_DEFAULT_FIELD(marshal_client_req_squid_len, DEFAULT_INT_FIELD)
+LOG_ACCESS_DEFAULT_FIELD(marshal_proxy_req_squid_len, DEFAULT_INT_FIELD)
+LOG_ACCESS_DEFAULT_FIELD(marshal_server_resp_squid_len, DEFAULT_INT_FIELD)
+LOG_ACCESS_DEFAULT_FIELD(marshal_cache_resp_squid_len, DEFAULT_INT_FIELD)
```

Comment:
I guess I'm ok with this, just seems a little odd and wildly different from all other similar code, no?